### ISSUE TITLE
SPOCC and other specimen APIs alignment

### ISSUE THREAD

**Issue by mjcollin** [idigbio-api-hackathon/HackathonCentral#15]
Integrate iDigBio APIs into SPOCC or evaluate another common API for all the various specimen record aggregators.

**Comment by sckott** [comment]
I assume you mean https://github.com/ropensci/spocc ?  If so, :+1:  and I can def. help here

**Comment by mjcollin** [comment]
Indeed. I noticed you wrote it and I'm glad you're coming :) Since you've worked on everyone else's APIs in some way, I'm looking forward to discussing some kind of end-game for them all. Is there enough commonality to what is stored and how it's accessed for occurrence aggregators & providers to settle on some standard behaviors like we've settled (kind of) on DwC for representing information?

**Comment by sckott** [comment]
There are some similarities, in addition to many differences.  wrt DwC, some of them have adopted DwC, including GBIF, VertNet, etc., but I don't think any prefix dwc terms as in idigbio `dwc:specificEpithet: "ilia"`  

Some pain points:
- Variation in spatial search - some only support WKT, some only bounding boxes, some points with radii, etc. 
- Taxonomic - each provider can have data for the same taxa, but unless they are following a particular taxonomy, we can run into problems in trying to use data from different providers for the same taxa. 
- Duplicates - alot of data eventually ends up in GBIF - so if using X provider in addition to GBIF, be careful
- the _I want all the data_ use case - rate limited REST APIs are fine for most people I think but some want e.g., **all the records**  available for 10K species.  This isn't a good fit for the average REST API, but better for a download API. Some providers provide programmatic access to download APIs (idigbio, vertnet, GBIF will soon)  - the downloaded data though requires different handling on the client side, but libraries should take care of this.

**Comment by sckott** [comment]
started adding in idigbio to `spocc`, see https://github.com/ropensci/spocc/issues/121

**Comment by sckott** [comment]
Got a few more things to work on, but essenitally working now
